src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -401,11 +398,8 @@ export default function ChartContainer<T extends ChartDataTypes>({
       !(blurTarget instanceof Element) ||
       !nodeBelongs(containerRefObject.current, blurTarget)
     ) {
-      setHighlightedPoint(null);
+      clearHighlightedSeries();
```

Comment:
Bar chart will not remember the highlighted item when tab out. However this is not consistent behavior between Bar, Line and Area charts.